Code/GraphMol/Wrap/rough_test.py

```diff
@@ -5738,6 +5738,64 @@ def heapPermutation(self, s = 0):
     self.assertIsNotNone(m)
     TestAssignChiralTypesFromMolParity(m, self)
 
+  def testRemoveHsParams(self):
+    smips = Chem.SmilesParserParams()
+    smips.removeHs = False
+
+    m = Chem.MolFromSmiles('F.[H]',smips)
+    ps = Chem.RemoveHsParameters()
```

Comment:
I think that these tests would all be clearer & more interesting if they used the default params first:

    m = Chem.MolFromSmiles('F.[H]',smips)
    ps = Chem.RemoveHsParameters()
    m = Chem.RemoveHs(m, ps)
    self.assertEqual(m.GetNumAtoms(),2)
    ps.removeDegreeZero = True
    m = Chem.RemoveHs(m,ps)
    self.assertEqual(m.GetNumAtoms(),1)

Reply:
yeah, I didn't do that since the default behavior is the same as it was before. But actually testing that is a good idea.